node/src/chain_spec/dancebox.rs

```diff
@@ -214,7 +214,7 @@ fn testnet_genesis(
     // Assign 1000 block credits to all container chains registered in genesis
     let para_id_credits: Vec<_> = para_ids
         .iter()
-        .map(|(para_id, _genesis_data, _boot_nodes)| (*para_id, 1000))
+        .map(|(para_id, _genesis_data, _boot_nodes)| (*para_id, 1000, 100))
```

Comment:
Might worth adding a comment here to not only explain the block production credits but also the collator assignment ones.